x-pack/plugins/lens/public/trigger_actions/open_lens_config/in_app_embeddable_edit/in_app_embeddable_edit_action_helpers.tsx

```diff
@@ -104,52 +105,55 @@ export async function executeEditEmbeddableAction({
     };
     onUpdate(newAttributes);
   };
+  core.theme.theme$.subscribe(async (theme) => {
+    const Component = await getEditLensConfiguration(core, deps, visualizationMap, datasourceMap);
+    const ConfigPanel = (
+      <EuiThemeProvider colorMode={theme.darkMode ? 'dark' : 'light'}>
```

Comment:
Is this for Lens to fix, or for us to fix?

Reply:
Ultimately it is for Lens to fix this as we don't own the code. [This comment](https://github.com/elastic/kibana/pull/179716#discussion_r1544880251) is aimed at the Lens team (or anyone feeling qualified to answer)

Reply:
Stratoula already mentioned this I think, what I mean is that we're not wrapping wherever we mount the Lens component in a EuiThemeProvider

Reply:
Yes we do https://github.com/elastic/kibana/blob/74386d037dc271b07ef46c1747c647360ac0df85/x-pack/plugins/observability_solution/observability_ai_assistant_app/public/utils/shared_providers.tsx

Reply:
I think we mount the edit lens component manually though.

Reply:
We do and possibly this can be fixed on the assistant too. But I am fine if the fix is applied to the lens side too. 👍